WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -61,6 +62,9 @@ public abstract class SendControlViewModel : WasabiDocumentTabViewModel, IWallet
 
 		private const string WaitingForHardwareWalletButtonTextString = "Waiting for Hardware Wallet...";
 
+		private static readonly FeeRate MinRelayTxFeeRate = new FeeRate(Money.Satoshis(1), 1);
+		private static readonly FeeRate AbsurdlyHighFeeRate = new FeeRate(Money.Satoshis(Constants.MaximumNumberOfSatoshis), 1);
```

Comment:
We should consider using a reasonable high fee rate not the maximum number of satoshis.
https://github.com/zkSNACKs/WalletWasabi/pull/3696#discussion_r471057297